public/javascripts/items.js

```diff
@@ -50,9 +50,12 @@ const descriptionEdited = event => {
 
 const completedClicked = event => {
   const element = $( event.target )
-  const id = element.data( 'id' )
+  const id = element.context.parentNode.parentNode.parentNode
```

Comment:
This is scary to me...  Is there a better way to identify this element? Minor changes in the HTML would break this... Could we provide a class or ID to the parent element we're interested in so that we can get at it more directly?